<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
  <class>MainWindow</class>
  <widget class="QMainWindow" name="MainWindow">
    <property name="geometry">
      <rect>
        <x>0</x>
        <y>0</y>
        <width>598</width>
        <height>395</height>
      </rect>
    </property>
    <property name="font">
      <font>
        <weight>50</weight>
        <bold>false</bold>
      </font>
    </property>
    <property name="windowTitle">
      <string>MaxIt</string>
    </property>
    <widget class="QWidget" name="centralWidget">
      <widget class="QTextEdit" name="first_score">
        <property name="geometry">
          <rect>
            <x>10</x>
            <y>0</y>
            <width>31</width>
            <height>31</height>
          </rect>
        </property>
      </widget>
      <widget class="QTextEdit" name="second_score">
        <property name="geometry">
          <rect>
            <x>310</x>
            <y>0</y>
            <width>31</width>
            <height>31</height>
          </rect>
        </property>
      </widget>
      <widget class="QPushButton" name="getSize">
        <property name="geometry">
          <rect>
            <x>10</x>
            <y>210</y>
            <width>121</width>
            <height>31</height>
          </rect>
        </property>
        <property name="text">
          <string>Ввести размер!</string>
        </property>
      </widget>
      <widget class="QTextEdit" name="game_status">
        <property name="geometry">
          <rect>
            <x>180</x>
            <y>210</y>
            <width>200</width>
            <height>20</height>
          </rect>
        </property>
      </widget>
      <widget class="QPushButton" name="saveGame">
        <property name="geometry">
          <rect>
            <x>270</x>
            <y>190</y>
            <width>80</width>
            <height>16</height>
          </rect>
        </property>
        <property name="text">
          <string>Сохранить игру</string>
        </property>
      </widget>
      <widget class="QPushButton" name="loadGame">
        <property name="geometry">
          <rect>
            <x>270</x>
            <y>170</y>
            <width>80</width>
            <height>16</height>
          </rect>
        </property>
        <property name="text">
          <string>Загрузить игру</string>
        </property>
      </widget>
    </widget>
    <widget class="QMenuBar" name="menuBar">
      <property name="geometry">
        <rect>
          <x>0</x>
          <y>0</y>
          <width>598</width>
          <height>17</height>
        </rect>
      </property>
      <widget class="QMenu" name="menu">
        <property name="title">
          <string>Меню</string>
        </property>
        <addaction name="exit_game"/>
        <addaction name="action"/>
      </widget>
      <addaction name="menu"/>
    </widget>
    <widget class="QToolBar" name="mainToolBar">
      <attribute name="toolBarArea">
        <enum>TopToolBarArea</enum>
      </attribute>
      <attribute name="toolBarBreak">
        <bool>false</bool>
      </attribute>
      <addaction name="new_new_game"/>
      <addaction name="exit_game"/>
    </widget>
    <widget class="QStatusBar" name="statusBar">
      <property name="enabled">
        <bool>true</bool>
      </property>
    </widget>
    <action name="start_new_game">
      <property name="icon">
        <iconset resource="res.qrc">
          <normaloff>:/new/prefix1/img/console.png</normaloff>:/new/prefix1/img/console.png
        </iconset>
      </property>
      <property name="text">
        <string>Новая игра</string>
      </property>
    </action>
    <action name="exit_game">
      <property name="icon">
        <iconset resource="res.qrc">
          <normaloff>:/new/prefix1/img/emergency-exit.png</normaloff>:/new/prefix1/img/emergency-exit.png
        </iconset>
      </property>
      <property name="text">
        <string>Завершить игру</string>
      </property>
    </action>
    <action name="new_new_game">
      <property name="icon">
        <iconset resource="res.qrc">
          <normaloff>:/new/prefix1/img/media-step-back-arrow.png</normaloff>:/new/prefix1/img/media-step-back-arrow.png
        </iconset>
      </property>
      <property name="text">
        <string>Ход назад</string>
      </property>
    </action>
    <action name="action">
      <property name="text">
        <string>Новая игра</string>
      </property>
    </action>
  </widget>
  <layoutdefault spacing="6" margin="11"/>
  <resources>
    <include location="res.qrc"/>
  </resources>
  <connections/>
</ui>
